### ISSUE TITLE
JUnit5 support: Use different ID for test engine

### ISSUE THREAD

**Issue by jlink** [TNG/ArchUnit#125]
When using ArchUnit with JUnit Platform 1.3.1 I get the following LOG:
```
WARNUNG: Third-party TestEngine implementations are forbidden to use the reserved 'junit-' prefix for their ID: 'junit-archunit'
```

So JUnit wants you to change your engine ID :-/

I suggest a plain `archunit` as do other engines, e.g. jqwik.net

**Comment by codecholeric** [comment]
I feel honored to say, that this warning was included in JUnit 5 just for me :wink: 
Originally I didn't know that there was a sort of namespace within engine ids.
Anyway, this should already be fixed in version [0.9.3](https://github.com/TNG/ArchUnit/releases/tag/v0.9.3), since I have changed the id after @sormuras pointed this out to me.

**Comment by jlink** [comment]
Actually I’m not sure a namespace is necessary here but we’re not the ones to decide here.
